### PR TITLE
Added a template for a good readme file

### PR DESCRIPTION
**Pull request by drdanjones**
Here's a starter for 10 for a readme file template. Thoughts?

### CODE REVIEW

File: README_template.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Installation
+A step by step guide on how to get your project working on a new machine.
+Note: Write this with someone else, using their machine. It is easy to forget
+dependencies.
+
+End this section with a simple example of how to get your project to do
+something useful.
+
+## Tests
```
Comment (RobinL) [inline README_template.md]: Can we add something on QA activities - i.e. a description of the QA activities which were conducted.  And a link to the AQA checklist `.md` file which they should have included (and which they can get from here: https://moj-analytical-services.github.io/our-coding-standards/web/)

File: README_template.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Explain how to run the tests for this program.
+### Automated Tests?
+If you have a set of automated tests how does a new user run them?
+### End-to-end tests
+What do these test? Why?
+
+## Authors
+A list of people who wrote the project.
+(Maybe a link to a list of contributors)
+
```
Comment (RobinL) [inline README_template.md]: Can we add a section on data - what data does the project use and where was it sourced from?

File: README_template.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Project Name
+One paragraph description of the project goes here.
+
```
Comment (RobinL) [inline README_template.md]: Add some text to say details should be included of who the proejct was for/ what it was used for.  Who was the primary customer?

**Comment by drdanjones** [comment]
Do those changes resolve your issues?

**Comment by EdwardAndress** [comment]
@RobinL - are you happy with these changes or is there more work to be done?

**Pull request merged**